<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_menu</class>
 <widget class="QWidget" name="main_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QLabel" name="gameTitle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Daggerfall</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newGameButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Новая игра</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadGameButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить игру</string>
   </property>
  </widget>
  <widget class="QLabel" name="bestResultLabel">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>562</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>01.01.01, Счёт 5, Последний враг: Каменный голем</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
